client/app/house/hooks/useCreateHouse.ts

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from 'next/navigation'
+import { useQueryClient } from '@tanstack/react-query'
+import { useForm } from 'react-hook-form'
+import { useToast } from '@/_hooks'
+import { isErrorWithMessage } from '@/_utils'
+import { useCreateHouseMutation } from './api'
+import { CreateHouseSchema, createHouseResolver } from '../schema'
+
+export const useCreateHouse = () => {
+  const queryClient = useQueryClient()
+  const router = useRouter()
+  const { toast } = useToast()
+
+  const { mutate, isPending } = useCreateHouseMutation()
+
+  const form = useForm<CreateHouseSchema>({
+    resolver: createHouseResolver,
+    defaultValues: {
+      name: '',
+      isExpensePerTime: undefined,
```

Comment:
I'm also going to have to fix this part in my task form, but the document says that undefined should not be defaultValues.
https://www.react-hook-form.com/api/useform/#defaultValues

Reply:
I'll fix it when I implement the edit form!